README.md

```diff
@@ -1 +1,4 @@
-# tic-tac-toe
\ No newline at end of file
+# Tic-Tac-Toe
+
+Install instructions:
++ install colorize: gem install colorize (mac & linux)
```

Comment:
Good :+1: 

Could you improve your README? The author's name, the description of the project or the link to.